AsyncDisplayKit/TextKit/ASTextKitTruncating.h

```diff
@@ -33,4 +33,9 @@
      truncationAttributedString:(NSAttributedString *)truncationAttributedString
          avoidTailTruncationSet:(NSCharacterSet *)avoidTailTruncationSet;
 
+/**
+ *  Actually do the truncation.
```

Comment:
Haha, yeah - let's leave out unnecessary comments, though humor is welcome in ones that also provide information to the reader :)